apps/chatbot/pyproject.toml

```diff
@@ -16,18 +17,16 @@ jupyterlab = "^4.4.0"
 langdetect = "^1.0.9"
 langfuse = "^2.60.2"
 llama-index = "^0.12.30"
-llama-index-embeddings-bedrock = "^0.5.0"
 llama-index-embeddings-google-genai = "^0.1.0"
-llama-index-llms-bedrock-converse = "^0.4.15"
 llama-index-llms-google-genai = "^0.1.7"
-llama-index-postprocessor-bedrock-rerank = "^0.3.2"
 llama-index-postprocessor-presidio = "^0.3.0"
 llama-index-storage-docstore-redis = "^0.3.0"
 llama-index-storage-index-store-redis = "^0.4.0"
 llama-index-vector-stores-redis = "^0.5.0"
 mangum = "^0.19.0"
 moto = { extras = ["cognitoidp"], version = "^5.1.0" }
 nh3 = "^0.2.21"
+pydantic-settings = "^2.10.1"
 pyjwt = "^2.10.1"
 pyproject-hooks = "^1.2.0"
 python = ">=3.12,<3.14"
```

Comment:
Can we remove from this file all dependencies that are not really required to run the lambda function?
I am just thinking about jupyter, gunicorn and probably more.
Or we want to do that in a dedicated PR?